ExtraDry/Sample.Server/Controllers/RegionController.cs

```diff
@@ -77,7 +77,7 @@ public async Task CreateAsync(Region value)
     [Authorize(SamplePolicies.SamplePolicy)]
     public async Task UpdateAsync(string code, Region item)
     {
-        await regions.UpdateAsync(code, item);
+        await regions.UpdateAsync(code, item, true);
```

Comment:
For readability and maintenance I usually spell out the bool flags on methods, e.g. `await regions.UpdateAsync(code, item, allowMove: true);`  No style guide here, just a suggestion.

Reply:
I've updated to do this as well. There's no style guide, but it will be copied and pasted from so might as well be explicit.